README.md

```diff
@@ -213,41 +215,67 @@ pam::manage_faillock_conf: true
 
 A couple of things to note here are:
 
+- This feature will only work on systems running EL 8 (or equivalent) and above.
 - ``pam::faillock`` must still be true for faillock to work appropriately
 - By default, /etc/security/faillock.conf will be empty except for a comment saying the file is managed by puppet. To set content in the file, the following parameters are available:
 
-  - ``pam::faillock_dir``
+  - ``pam::faillock_log_dir``
   - ``pam::faillock_audit``
-  - ``pam::faillock_silent``
+  - ``pam::display_account_lock``
   - ``pam::faillock_no_log_info``
   - ``pam::faillock_local_users_only``
   - ``pam::faillock_nodelay``
-  - ``pam::faillock_deny``
-  - ``pam::faillock_fail_interval``
-  - ``pam::faillock_unlock_time``
-  - ``pam::faillock_even_deny_root``
-  - ``pam::faillock_root_unlock_time``
+  - ``pam::deny``
+  - ``pam::fail_interval``
+  - ``pam::unlock_time``
+  - ``pam::even_deny_root``
+  - ``pam::root_unlock_time``
   - ``pam::faillock_admin_group``
 
 #### /etc/security/faillock.conf Hieradata Example With All Parameters
 
 ```yaml
 pam::faillock: true
 pam::manage_faillock_conf: true
-pam::faillock_dir: '/var/log/faillock'
+pam::faillock_log_dir: '/var/log/faillock'
 pam::faillock_audit: true
-pam::faillock_silent: true
+pam::display_account_lock: true
 pam::faillock_no_log_info: false
 pam::faillock_local_users_only: false
 pam::faillock_nodelay: false
-pam::faillock_deny: 5
-pam::faillock_fail_interval: 900
-pam::faillock_unlock_time: 900
-pam::faillock_even_deny_root: true
-pam::faillock_root_unlock_time: 60
+pam::deny: 5
+pam::fail_interval: 900
+pam::unlock_time: 900
+pam::even_deny_root: true
+pam::root_unlock_time: 60
 pam::faillock_admin_group: 'wheel'
 ```
 
+### Managing /etc/security/pwhistory.conf
+
+To manage pwhistory with ``/etc/security/pwhistory.conf`` instead of inline parameters in the auth files set the following in hieradata:
```

Comment:
I'd drop the text `instead of inline parameters in the auth files`.

Also shouldn't those be single backticks, not double?  (Double backticks render as single backticks, IIRC, not as a monospace font block.)

Reply:
The double backticks are all over the place in the readme and they appear to render without issues. That said, I'll go through and try to be consistent with the number of backticks being used.